scaleway/resource_instance_server_test.go

```diff
@@ -360,14 +356,12 @@ func TestAccScalewayInstanceServer_UserData_WithoutCloudInitAtStart(t *testing.T
 			{
 				// Without cloud-init
 				Config: `
-					data "scaleway_instance_image" "ubuntu" {
-						architecture = "x86_64"
-						name         = "Ubuntu 20.04 Focal Fossa"
-					}
-
 					resource "scaleway_instance_server" "base" {
-						image = "${data.scaleway_instance_image.ubuntu.id}"
+						image = "ubuntu-focal"
```

Comment:
is it normal that it is `ubuntu-focal` here and `ubuntu_focal` down there ?

Reply:
The label on instance server supports the use of '-' and '_'

Reply:
I suggest using only one everywhere. By doing so, it is easier to migrate all occurrences to the next Ubuntu LTS once it is made available.